Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -89,7 +128,7 @@ def render_GET(self, request):
             neighbor_level = int(request.args["neighbor_level"][0])
 
         # TODO: Remove dummy return and change to aggregated data calculation
```

Comment:
I'd suggest leaving it, as Yinghao will be moving this data in his pull request either way, which will be due shortly